src/routes/user/verifyToken.js

```diff
@@ -2,16 +2,16 @@ import { StatusCodes } from 'http-status-codes';
 import jsonwebtoken from 'jsonwebtoken';
 
 const auth = (req, res, next) => {
```

Comment:
I think that you should create a new folder `/src/middlewares/verifyToken.middleware.js`